packages/action-bar/src/spectrum-action-bar.css

```diff
@@ -13,102 +13,215 @@ governing permissions and limitations under the License.
+        var(--spectrum-actionbar-item-counter-color)
+    );
+    font-size: var(
+        --mod-actionbar-item-counter-font-size,
+        var(--spectrum-actionbar-item-counter-font-size)
+    );
+    line-height: var(
+        --mod-actionbar-item-counter-line-height,
+        var(--spectrum-actionbar-item-counter-line-height)
+    );
+    margin-block-start: var(
+        --mod-actionbar-spacing-item-counter-top,
+        var(--spectrum-actionbar-spacing-item-counter-top)
+    );
+    margin-inline-end: var(
+        --mod-actionbar-spacing-item-counter-end,
+        var(--spectrum-actionbar-spacing-item-counter-end)
+    ); /* .spectrum-ActionBar .spectrum-FieldLabel */
+    padding: 0;
 }
-:host([flexible]) #popover {
-    width: auto; /* .spectrum-ActionBar--flexible .spectrum-ActionBar-popover */
+::slotted(sp-field-label):lang(ja),
+::slotted(sp-field-label):lang(ko),
+::slotted(sp-field-label):lang(zh) {
+    line-height: var(
+        --mod-actionbar-item-counter-line-height-cjk,
+        var(--spectrum-actionbar-item-counter-line-height-cjk)
+    ); /* .spectrum-ActionBar .spectrum-FieldLabel:lang(ja),
+   * .spectrum-ActionBar .spectrum-FieldLabel:lang(zh),
+   * .spectrum-ActionBar .spectrum-FieldLabel:lang(ko) */
 }
-:host([variant='fixed']) {
-    position: fixed; /* .spectrum-ActionBar--fixed */
+::slotted(sp-action-group) {
+    margin-block-start: var(
+        --mod-actionbar-spacing-action-group-top,
+        var(--spectrum-actionbar-spacing-action-group-top)
+    );
+    margin-inline-end: var(
+        --mod-actionbar-spacing-action-group-end,
+        var(--spectrum-actionbar-spacing-action-group-end)
+    ); /* .spectrum-ActionBar .spectrum-ActionGroup */
+    margin-inline-start: auto;
 }
-:host([dir='ltr']) #popover {
-    padding-left: var(
-        --spectrum-actionbar-padding-left
-    ); /* [dir=ltr] .spectrum-ActionBar-popover */
+:host([emphasized]) #popover {
+    background-color: var(
+        --mod-actionbar-emphasized-background-color,
+        var(--spectrum-actionbar-emphasized-background-color)
+    );
+    border-color: transparent;
+    filter: none; /* .spectrum-ActionBar--emphasized .spectrum-ActionBar-popover */
 }
-:host([dir='rtl']) #popover {
-    padding-right: var(
-        --spectrum-actionbar-padding-left
-    ); /* [dir=rtl] .spectrum-ActionBar-popover */
+:host([emphasized]) .spectrum-ActionButton,
+:host([emphasized]) .spectrum-CloseButton-UIIcon,
+:host([emphasized]) ::slotted(sp-field-label) {
+    fill: var(
+        --mod-actionbar-emphasized-item-counter-color,
+        var(--spectrum-actionbar-emphasized-item-counter-color)
+    );
+    color: var(
+        --mod-actionbar-emphasized-item-counter-color,
+        var(--spectrum-actionbar-emphasized-item-counter-color)
+    ); /* .spectrum-ActionBar--emphasized .spectrum-ActionButton,
+   * .spectrum-ActionBar--emphasized .spectrum-FieldLabel,
+   * .spectrum-ActionBar--emphasized .spectrum-CloseButton-UIIcon */
```

Comment:
@pfulton and @lunarfusion this would otherwise make these buttons "static white", right?

Does this point at the benefit of asking the design team to include a "static white" Field Label?

Reply:
@pfulton if this "static white"s the Buttons, should we look at having Action Group manage the `variant` of child buttons the way you added support for managing `size` that way?